pywebpush/__init__.py

```diff
@@ -86,6 +86,11 @@ def __init__(self, subscription_info):
             the client.
 
         """
+        # Python 2 v. 3 hack
```

Comment:
I'd recommend using the six library rather than custom hacks:
https://pythonhosted.org/six/#binary-and-text-data